Projects/Embark/Sources/Viewables/Tooltip+Alert.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  EmbarkAlert.swift
+//  Embark
+//
+//  Created by Tarik Stafford on 2021-02-05.
+//  Copyright © 2021 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import hCore
+import hCoreUI
+import Presentation
+import UIKit
+import hGraphQL
+
+typealias Tooltip = GraphQL.EmbarkStoryQuery.Data.EmbarkStory.Passage.Tooltip
+
+extension Tooltip: Presentable {
+    public func materialize() -> (UIViewController, Future<Void>)  {
+        
+        let containerView = UIStackView()
+        containerView.layoutMargins = UIEdgeInsets(horizontalInset: 32, verticalInset: 20)
+        containerView.isLayoutMarginsRelativeArrangement = true
+        containerView.axis = .vertical
+        
+        let viewController = UIViewController()
+        viewController.view.backgroundColor = .white
+        let bag = DisposeBag()
+        
+        viewController.view.addSubview(containerView)
+        containerView.snp.makeConstraints { (make) in
+            make.leading.trailing.top.equalToSuperview()
+            make.bottom.equalToSuperview().inset(viewController.view.safeAreaInsets.bottom)
+        }
+        
+        viewController.title = title
+        
+        bag += containerView.didLayoutSignal.onValue({ (_) in
+            viewController.preferredContentSize = containerView.systemLayoutSizeFitting(.zero)
+        })
+        
+        return (viewController, Future { completion in
+            
```

Comment:
NIT: Don't start with newline

Reply:
I always find it much more readable personally. It's like having a title for a chapter. It makes the title harder to read if the paragraph starts right underneath :)